spotify-controller@koolskateguy89/prefs.js

```diff
@@ -8,7 +8,7 @@ const Config = imports.misc.config;
 const [major] = Config.PACKAGE_VERSION.split('.');
 const shellVersion = Number.parseInt(major);
 
-if (shellVersion < 40) {
+if (shellVersion < 42) {
```

Comment:
The reason this is `40` is because Gnome 40 introduced major changes. Checking the shell version is before Gnome 42 will break this for Gnome 40 & 41.